components/03-modules/products-grid-item/_products-grid-item.scss

```diff
@@ -1,90 +1,106 @@
-$products-grid-item__action-item-size                 : 32px !default;
-$products-grid-item__action-icon-size                 : 32px !default;
-$products-grid-item__action-item-spacing              : $spacer !default;
-$products-grid-item__badge-top-position               : $spacer--medium !default;
-$products-grid-item__badge-left-position              : 0 !default;
 $products-grid-item__base-spacing                     : $spacer !default;
-$products-grid-item__content-bg-color                 : $white !default;
 $products-grid-item__padding                          : 0 !default;
-$products-grid-item__text-padding                     : 0 $spacer !default;
 $products-grid-item__border                           : 1px solid $gray-lightest !default;
-$products-grid-item__font-size-medium                 : $font-size-small !default;
-$products-grid-item__font-size-large                  : $font-size-base !default;
-$products-grid-item__content-padding                  : 22px $spacer--medium 12px !default;
+
+$products-grid-item__font-size                        : $font-size-small !default;
+$products-grid-item__font-size\@large                 : $font-size-base !default;
+$products-grid-item__line-height                      : 1.5 !default;
+
+$products-grid-item__badge-left-position              : 0 !default;
+$products-grid-item__badge-top-position               : $spacer--medium !default;
+
+$products-grid-item__content-bg-color                 : $white !default;
+$products-grid-item__content-padding                  : $spacer !default;
+$products-grid-item__content-padding\@medium          : $spacer--medium !default;
+
 $products-grid-item__name-color                       : $font-color-base !default;
 $products-grid-item__name-font-weight                 : $font-weight-normal !default;
-$products-grid-item__name-margin-bottom               : 12px !default;
-$products-grid-item__price-base-color                 : $bg-color-dark !default;
+
+
+$products-grid-item__action-size                      : 32px !default;
+$products-grid-item__action-item-spacing              : $spacer !default;
 
 .products-grid-item {
-    max-width: 320px; // for proper imageHeight calculation based on container width
+    position: relative;
     display: block;
     padding: $products-grid-item__padding;
     border: $products-grid-item__border;
-    position: relative;
     overflow: hidden;
 
+    &:hover {
+        .products-grid-item__details {
+            transform: translateY(-100%);
+        }
+    }
```

Comment:
When we will resize window to mobile (around 320px) and see hover effect we see that product name has different width for default state and for the state when second section goes up. 

Also the effect imho  isn't the best. Would be great if we make apear duplicated section on top of current and start translate from that point. or maybe from bottom but ad some kind fadeIn effect?

Reply:
Right, isn't best, but it's good enough as a MVP and we can work on this later.

The name size is different on purpose, to prevent long names take a lot of vertical space (design unfortunately doesn't describe how this should work).